widget/list_test.go

```diff
@@ -161,10 +161,74 @@ func TestList_SmallList(t *testing.T) {
 			item.(*fyne.Container).Objects[1].(*Label).SetText(data[index])
 		},
 	)
-	list.Resize(fyne.NewSize(200, 1000))
+	w := test.NewWindow(list)
+	w.Resize(fyne.NewSize(200, 1000))
```

Comment:
Maybe these images could be smaller? that's a lot of empty space

Reply:
sure

Reply:
Resolved